app/vmalert/main.go

```diff
@@ -117,6 +122,16 @@ func main() {
 		logger.Fatalf("failed to parse %q: %s", *ruleTemplatesPath, err)
 	}
 
+	if *unitTest {
+		if len(*unitTestFiles) == 0 {
+			logger.Panicf("should set at lease one file in --unittest.file for test")
+		}
```

Comment:
```suggestion
	if len(*unitTestFiles) > 0 {{
```
can we do this instead of having two flags for unittests?

Reply:
Done!